README.md

```diff
@@ -0,0 +1,568 @@
+                tocfeed
+                epub
+                enhancedchm
+```
+
+
+> **_NOTE:_**
+> The format packages are provided by separate PEAR packages
+> (doc.php.net/PhD_Generic, doc.php.net/PhD_IDE, doc.php.net/PhD_PEAR
+> and doc.php.net/PhD_PHP) where only the Generic is installed by
+> default.
+
+You can tell by the output of the `--list` option that PhD can also be
+used to render the docs as a PDF file or as Unix Man Pages.
+
+To select a format and package, you must use the `-f [formatName]` and
+`-P [packageName]` options.
+
+```shell
+$ phd -f manpage -P PHP -d .manual.xml
+```
+
+This command will output the documentation for PHP functions in the Unix
+Man page format.
+
+## Compiling the PhD guide
+
+The PhD guide is this manual you are reading. It lives in
+PhD's Git repository under `docs/phd-guide/phd-guide.xml`. If you
+installed PhD from Git, you already have it. Otherwise, get it:
+
+```shell
+$ svn checkout http://svn.php.net:/repository/phd/trunk/docs/phd-guide
+U phd/docs/phd-guide/phd-guide.xml
+```
+
+Now you have everything you need. Just type
+
+```shell
+$ cd phd/docs/phd-guide/
+$ phd -f bigxhtml -d phd-guide.xml
+```
+
+There should be an .html file in the directory now. View it with a
+browser!
+
+## Customizing the rendering results
+
+PhD lets you specify a number of options to customize the generated
+documentation files. The following sections describe some of them.
+
+### Source code highlighter
+
+Part of the documentation of programming languages is source code
+examples. PhD is able to colorize the source code of many types of
+source code with the help of *highlighters*.
+
+To utilize syntax highlighting, your opening `<programlisting>` tags
+need a `role` attribute describing the type of source code. Examples are
+`php`, `html` and `python`.
+
+> **_NOTE:_**
+> PhD currently only highlights the code if it is embedded in a `CDATA`
+> section.
+
+```xml
+<programlisting role="php"><![CDATA[
+<?php
+echo "Hello world!";
+?>
+]]></programlisting>
+```
+
+By default, PhD uses the source code highlighter that is built into PHP
+itself which is only able to highlight PHP code.
+
+If your documentation contains other types of source code or markup,
+like XML, HTML, Javascript or any other language, you should try the
+[GeSHi](http://qbnz.com/highlighter/) highlighter that is shipped with
+PhD:
+
+1.  Install GeSHi from the MediaWiki PEAR channel:
+
+```shell
+$ pear channel-discover mediawiki.googlecode.com/svn
+$ pear install mediawiki/geshi
+```
+
+2.  Use the GeSHi syntax highlighting class when rendering your
+    documentation:
+
+```shell
+$ phd -g 'phpdotnet\phd\Highlighter_GeSHi' -d phd-guide.xml
+```
+
+If you have GeSHi version 1.1.x installed, you should use the
+`phpdotnet\phd\Highlighter_GeSHi11x` highlighter, which is adapted to
+GeSHi's new API.
+
+Apart from using the highlighter shipped with PhD, you can [build your
+own highlighters](#writing-a-custom-syntax-highlighter).
```

Comment:
Does this even still work ?

Reply:
This doesn't work without PEAR so I think we should remove this too.